openstudiocore/src/utilities/filetypes/EpwFile.cpp

```diff
@@ -98,17 +98,16 @@ namespace openstudio{
     return fp*psat;
   }
 
-  // TODO: Warning -Wunused-function
-  static double enthalpy(double T, double p, double phi)
+  // Note JM 2019-01-03: suppress -Wunused-function warning for this API function
+  [[maybe_unused]] static double enthalpy(double T, double p, double phi)
```

Comment:
It's not in header. It's a free standing function, that's not being called in AirState ctor unlike the other freestanding functions, though AirState does the same as these functions so it could call them (search for 'eqn 22' for eg)